NEWS.md

```diff
@@ -1,2 +1,16 @@
+## 2018-08-30 v1.0.0-SNAPSHOT
+The module provide storing of templates and access to them through REST API. 
+Templates support internationalization and store a set of templates texts for each language. 
+Also templates contain meta information that describes the type of source document and its scope.
+
+CRUD API for templates was added: 
+
+| METHOD                             | DESCRIPTION                                        |
+|------------------------------------|----------------------------------------------------|
+| POST /template                     | Create new template in storage                     |
+| GET /template/id                   | Get template from storage by id                    |
```

Comment:
I would suggest to change it to 'GET /template/{templateId}' to be more consistent and emphasize that templateId is changeable parameter